packages/dev/docs/pages/react-aria/ssr.mdx

```diff
@@ -57,26 +61,21 @@ function MyComponent() {
 }
 ```
 
-## Locale selection
+## Automatic ID Generation
 
-When using server side rendering, the application should be wrapped in an [I18nProvider](I18nProvider.html) with an explicit `locale` prop, rather than relying on automatic locale selection. This ensures that the locale of the content rendered on the server matches the locale expected by the browser. The `Accept-Language` HTTP header, which the browser sends to the server with the user’s desired language, could be used to implement this. You could also use an in-application setting for this if available, or locale-specific URLs, for example. In addition to passing the `locale` prop to the `I18nProvider`, you should also ensure the `lang` and `dir` attributes are set on the `<html>` element for your page.
+When using SSR, only a single copy of React Aria can be on the page at a time. This is in contrast to client-side rendering, where multiple copies from different parts of an app can coexist. Internally, many components rely on auto-generated ids to link related elements via ARIA attributes. These ids typically use a randomly generated seed plus an incrementing counter to ensure uniqueness even when multiple instances of React Aria are on the page. With SSR, we need to ensure that these ids are consistent between the server and client.  This means the counter resets on every request, and we use a consistent seed. Due to this, multiple copies of React Aria cannot be supported because the auto-generated ids would conflict.
```

Comment:
do we really want to explicitly say this?
> This is in contrast to client-side rendering, where multiple copies from different parts of an app can coexist.

I think we should just omit this. Or say, you need to be extra careful ensuring only one copy for SSR, with the implication it's better to only have one anyways

Reply:
this has always been there (it's been in production for years), it just moved down.